test-cases/tests/performance/k02-run-performance-test-against-3scale.md

```diff
@@ -14,6 +14,7 @@ products:
     targets:
       - 0.1.0
       - 0.2.0
+      - 1.0.0
```

Comment:
does this need to be split?

Reply:
good point, actually there's a [separate test case for rhoam](https://github.com/integr8ly/integreatly-operator/blob/master/test-cases/tests/performance/k04-run-performance-test-against-rhoam-3scale-user-sso.md)

I'll remove the rhoam test case metadata from this file